src/pages/RegistrationForm.tsx

```diff
@@ -70,6 +72,11 @@ function RegistrationForm({ isStudent }: { isStudent: boolean }) {
         loading={loading}
         disabled={loading}
         submitWithoutChange={isRegistering}
+        staticMessage={
+          isStudent
+            ? "Students cannot register as mentors"
```

Comment:
```suggestion
            ? "NOTE: Students cannot register as mentors"
```

This would look better.

Reply:
Done
![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/0577a615-eff4-45a3-9197-ee2fcd18de60)

![image](https://github.com/kossiitkgp/KWoC-Frontend/assets/64408989/d208c25c-9b25-49a6-9258-4ed1a3f70e24)